main_courses/hard/pierogi.md

```diff
@@ -0,0 +1,48 @@
+# Pierogi
+
+## Dough 
+
+### Ingredients
+
+* 0.5kg plain flour
+* 1 large eggs
+* 1 glass of lukewarm water
+* 1 tbsp of oil
+* Pinch of salt
+
+### Method
+
+To make the dough:
+1. Mix flour and salt in a bowl/on a kneading board (as for pasta).
+2. Make a well in the centre and add the eggs and oil into it.
+3. Mix together, slowly adding water, until it forms a smooth and non-sticky dough.
+4. Leave to rest, covered by a clean tea towel or cling film, for 30mins.
+
+To prepare the pierogi:
+1. Split the dough in half; leave one half covered and roll out the second half on a well-floured surface. You want the dough to be fairly thin, around ~2-3mm of thickness.
+2. Using the rim of a floured drinking glass, cut out circles in the rolled-out dough.
+3. Picking up one circle at a time, put in the filling - 1-2 teaspoons, depending on your confidence -, fold the circle in half and crimp the edges shut. 
+4. Repeat until all the pierogi are prepared. Keep the finished pierogi on a floured baking sheet and covered so they don't dry out. 
+
+To cook the pierogi: 
+1. Working in batches, put the pierogi in boiling water. Reduce to a simmer, and cook for a few minutes; once they start rising to the surface, check for doneness. Remove with a slotted spoon once done.  
+
+To serve:
+The cooked pierogi can be eaten or reheated as they - or you can pan-fry them in a bit of oil and serve them with some caramelized onions, fried bacon bits, and some sour cream. 
```

Comment:
I think you're missing a word after 'they'?

Reply:
```suggestion
The cooked pierogi can be eaten or reheated as they are - or you can pan-fry them in a bit of oil and serve them with some caramelized onions, fried bacon bits, and some sour cream. 
```
There is indeed a word missing.